stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/ClientState.kt

```diff
@@ -80,11 +84,21 @@ class ClientState(client: StreamVideo) {
 
     internal val clientImpl = client as StreamVideoImpl
 
+    /**
+     * Returns true if there is an active or ringing call
+     */
+    fun hasActiveOrRingingCall(): Boolean = safeCall(false) {
+        val hasActiveCall = _activeCall.value != null
+        val hasRingingCall = _ringingCall.value != null
+        val activeOrRingingCall = hasActiveCall || hasRingingCall
+        logger.d { "[hasActiveOrRingingCall] active: $hasActiveCall, ringing: $hasRingingCall" }
+        activeOrRingingCall
+    }
```

Comment:
Because we are accessing the state flows which can crash, I don't want to have a case where the flow is unavailable and it crashes, just want to return false in that case.